app/views/customers/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Customer: <%= @customer.name %></h1>
+<p>
+<%= @customer.name %>'s Items:<br>
+<% @customer.items.each do |item| %>
+  <%= item.name %>, Price: $<%= item.price %>, Supermarket: <%= item.supermarket.name %><br>
+<% end %>
+</p>
+<%= form_with url: "/customers/#{@customer.id}", method: :patch, data: {turbo: false} do |form| %>
```

Comment:
Per my comment in your `customers` controller, to make this route more RESTful and keep our `customers#update` open to truly update a customer we could add
```ruby
"/customers/#{@customer.id}/items"
```
This will allow you to create this route:
```ruby
post "/customers/:customer_id/items", to: "customer_items#create"
```
Now we can utilize our join table to add an item to a customer by creating that association.